mobile/lib/features/stable/bitcoinize_confirmation_sheet.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:flutter/material.dart';
+import 'package:get_10101/common/value_data_row.dart';
+import 'package:get_10101/features/stable/stable_dialog.dart';
+import 'package:get_10101/features/stable/stable_value_change_notifier.dart';
+import 'package:get_10101/features/trade/domain/direction.dart';
+import 'package:get_10101/features/trade/domain/position.dart';
+import 'package:get_10101/features/trade/submit_order_change_notifier.dart';
+import 'package:go_router/go_router.dart';
+import 'package:provider/provider.dart';
+
+bitcoinizeBottomSheet({required BuildContext context, required Position position}) {
+  showModalBottomSheet<void>(
+    shape: const RoundedRectangleBorder(
+      borderRadius: BorderRadius.vertical(
+        top: Radius.circular(20),
+      ),
+    ),
+    clipBehavior: Clip.antiAlias,
+    isScrollControlled: true,
+    useRootNavigator: false,
+    context: context,
+    builder: (BuildContext context) {
+      return SafeArea(
+          child: Padding(
+        // padding: MediaQuery.of(context).viewInsets,
+        padding: EdgeInsets.only(bottom: MediaQuery.of(context).viewInsets.bottom),
+        // the GestureDetector ensures that we can close the keyboard by tapping into the modal
+        child: GestureDetector(
+          onTap: () {
+            FocusScopeNode currentFocus = FocusScope.of(context);
+
+            if (!currentFocus.hasPrimaryFocus) {
+              currentFocus.unfocus();
+            }
+          },
+          child: SingleChildScrollView(
+            child: SizedBox(
+              // TODO: Find a way to make height dynamic depending on the children size
+              // This is needed because otherwise the keyboard does not push the sheet up correctly
+              height: 300,
+              child: BitcoinizeBottomSheet(position: position),
+            ),
+          ),
+        ),
+      ));
+    },
+  );
+}
+
+class BitcoinizeBottomSheet extends StatelessWidget {
+  final Position position;
+
+  const BitcoinizeBottomSheet({super.key, required this.position});
+
+  @override
+  Widget build(BuildContext context) {
+    final stableValuesChangeNotifier = context.watch<StableValuesChangeNotifier>();
+    final submitOrderChangeNotifier = context.watch<SubmitOrderChangeNotifier>();
+
+    final stableValues = stableValuesChangeNotifier.stableValues();
+    stableValues.quantity = position.quantity;
+    stableValues.direction = Direction.long;
+
+    if (submitOrderChangeNotifier.pendingOrder != null &&
+        submitOrderChangeNotifier.pendingOrder!.state == PendingOrderState.submitting) {
+      final pendingOrder = submitOrderChangeNotifier.pendingOrder;
+
+      WidgetsBinding.instance.addPostFrameCallback((_) async {
+        return await showDialog(
+            context: context,
+            useRootNavigator: false,
```

Comment:
Note: we will likely need to change this again once https://github.com/get10101/10101/pull/1144 lands.